README.md

```diff
@@ -346,6 +346,21 @@ used to retrieve label/value attributes for the `option` tags. Besides that, you
   for the _optgroup_ (**Simple Form** will attempt to guess the best one the same way it does with
   `:label_method`)
 
+`:select` input accepts `:selected` option, which can be a single value, an array of values, or a lambda/procs, like:
+
+```ruby
+# Simplest example
+# Note that the value passed in `selected` is an id, not an object
+f.input :parent_id, collection: Parent.all, selected: child.parent_id
+# "Multiple" select
+# Note that `:multiple` is passed inside `:input_html`
+f.input :children_ids, collection: Child.all, selected: parent.children_ids, input_html: {multiple: true}
+# Proc/Lambda
+# The value of `selected` is passed into rails helper without manupulation, so check its doc/code for detail
+f.input :caught_pokemon_type_ids, collection: Pokemon::Type.all, 
+  selected: lambda {|type| trainer.caught_pokemons.map(&:type_id)}
+```
```

Comment:
Thanks, but I think we don't need to show all possibilities here, since this is actually a functionality provided by Rails. I think it might make more sense to extend the prompt example on line 326, with something like this:

    All other options given are sent straight to the underlying helper. For example, you can give prompt as:

    ```ruby
    f.input :age, collection: 18..60, prompt: "Select your age"
    ```

    Or you can set the selected option as well:

    ```ruby
    f.input :age, collection: 18..60, selected: 21
    ```

    You can check the `collection_select` Rails documentation for more info on the available options.
```

And this last part that points to collection select rails docs could be a link to the api or something. Wdyt?